windows/internal/driver_update.bat

```diff
@@ -1,8 +1,8 @@
-set "DRIVER_DOWNLOAD_LINK=https://s3.amazonaws.com/ossci-windows/451.82-tesla-desktop-winserver-2019-2016-international.exe"
-curl --retry 3 -kL %DRIVER_DOWNLOAD_LINK% --output 451.82-tesla-desktop-winserver-2019-2016-international.exe
+set "DRIVER_DOWNLOAD_LINK=https://ossci-windows.s3.amazonaws.com/452.39-data-center-tesla-desktop-win10-64bit-international.exe"
+curl --retry 3 -kL %DRIVER_DOWNLOAD_LINK% --output 452.39-data-center-tesla-desktop-win10-64bit-international.exe
```

Comment:
@peterjc123 Looks like this version is working? https://github.com/pytorch/pytorch/pull/48469#issuecomment-734108204

Reply:
Nope. Actually we are using the driver in the CUDA package.

Reply:
Oh, I see, it is at https://github.com/pytorch/pytorch/pull/48469/files#diff-54963e0b8298be39f2fb1705ffd5b9ef4043ad70226ee3a837f6d5e80aa53d74L339

Reply:
Would you please change that?

Reply:
Changed in https://github.com/pytorch/builder/pull/574/files#diff-59686e8c8da2a64d7f4ddebde6c4e5a0231215eda025e43e4f3311a4ceff27d3